src/examples/avatar/AvatarShape.tsx

```diff
@@ -8,23 +8,23 @@
 import React from 'react';
 import { PALETTE } from '@zendeskgarden/react-theming';
 import { Avatar } from '@zendeskgarden/react-avatars';
-import { Row, Col } from '@zendeskgarden/react-grid';
+import { Grid } from '@zendeskgarden/react-grid';
 import { ReactComponent as ZendeskIcon } from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
 import { ReactComponent as UserIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
 
 const Example = () => (
-  <Row>
-    <Col textAlign="center">
+  <Grid.Row>
+    <Grid.Col textAlign="center">
       <Avatar backgroundColor={PALETTE.grey[700]} size="medium">
```

Comment:
Is it worth demonstrating shade changes between light/dark mode in an example (using `useTheme`)?

While design doesn't necessarily require it universally, I wonder if that might help folks get a quick sense of how it might work given design's current avatar light/dark examples seem to swap icon color + lower the shade a level in dark mode:

<img width="138" alt="Screenshot 2024-08-13 at 4 20 47 PM" src="https://github.com/user-attachments/assets/939c2f2d-912b-47d8-9318-f01e8f715de3">
<img width="134" alt="Screenshot 2024-08-13 at 4 20 55 PM" src="https://github.com/user-attachments/assets/3a01e1a9-ff21-4da9-9b58-f950e9f9d9e5">

I would also concede such a demonstration could be best suited over in the ThemeProvider examples. :)

Reply:
Good point. `Avatar` is a very versatile component and often requires user-defined colors. To follow the visual treatment found in Figma for "User Avatars", devs would need to wire it as such:

```
<Avatar
  backgroundColor={getColor({
    theme,
    hue: 'neutralHue',
    light: { shade: 700 },
    dark: { shade: 600 }
  })}
  foregroundColor={getColor({ theme, variable: 'foreground.onEmphasis' })}
  
  ...
>
  <UserIcon role="img" aria-label="User" />
</Avatar>
```

We could drive that point home by replacing all examples featuring a "User Avatar" with the code above.

What do you think?

Reply:
Once the v9 functionality is available, I'd recommend using `<Avatar backgroundColor="background.emphasis" foregroundColor="foreground.onEmphasis" />`